lib/std/crypto/25519/curve25519.zig

```diff
@@ -28,14 +29,14 @@ pub const Curve25519 = struct {
     pub const basePoint = Curve25519{ .x = Fe.curve25519BasePoint };
 
     /// Check that the encoding of a Curve25519 point is canonical.
-    pub fn rejectNonCanonical(s: [32]u8) !void {
+    pub fn rejectNonCanonical(s: [32]u8) Error!void {
         return Fe.rejectNonCanonical(s, false);
     }
 
     /// Reject the neutral element.
-    pub fn rejectIdentity(p: Curve25519) !void {
+    pub fn rejectIdentity(p: Curve25519) Error!void {
         if (p.x.isZero()) {
-            return error.IdentityElement;
+            return Error.IdentityElement;
```

Comment:
I don't think this change is required since `error.IdentityElement == Error.IdentityElement` by definition of the [error set type](https://ziglang.org/documentation/0.7.1/#toc-Error-Set-Type), i.e., you can declare the same error value in multiple error sets but the same values will always get assigned the same underlying `u16` values.

Reply:
ZLS can complete `Error`, not `error` :)

But sure, lemme change all the return statements back to `error`.